### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by piyushsh25**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,48 +1,181 @@
...
+    padding: var(--size20);
+    background: var(--color-success);
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+@media only screen and (max-width:940px) {
```
[the remark this answers is not in the document]
Comment (piyushsh25) [reply index.css]: done!

File: index.html [hunk 2]
```diff
@@ -6,34 +6,181 @@
...
 
 <body>
-    <div>
+    <div class="main-div">
         <div class="nav-container">
             <div class="header">
                 <div class="title-logo"><i class="fab fa-deezer"></i></div>
                 <div class="title">
-                    Zevnom
+                    <a href="">Zevnon</a>
```
Comment (Hamdrive) [inline index.html:22]: Add a href value, else since it is for homepage atleast either add its path or #

File: index.html [hunk 3]
```diff
@@ -6,34 +6,181 @@
...
                 </div>
-                <input type="text" class="input" placeholder="search zenvon">
-
+                <input type="text" class="input" placeholder="search zevnom">
```
Comment (Hamdrive) [inline index.html:24]: typo in ecom name

File: index.html [hunk 4]
```diff
@@ -6,34 +6,181 @@
...
+                    <li class="items items-signup">Signup</li>
+
+                    <li class="items">
+                        <div class="avatar-badge md"> <i class="fas fa-heart"></i>
+                            <div class="badge text">0</div>
+                        </div>
+                    </li>
+                    <li class="items ">
+                        <div class="avatar-badge md"><i class="fas fa-shopping-cart"></i>
+                            <div class="badge text">0</div>
```
Comment (Hamdrive) [inline index.html:38]: Use a code formatter, makes it easier to read code

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Code looks good, apart from requested changes.

File: index.html (hunk 4 above)
Comment (piyushsh25) [reply index.html:38]: done!

File: index.html (hunk 3 above)
Comment (piyushsh25) [reply index.html]: its zevnon. I fixed the typo in my entire document. I will fix it in netlify too.

File: index.html (hunk 2 above)
Comment (piyushsh25) [reply index.html]: Done.

**Pull request merged**